src/corona_hakab_model/project_structure.py

```diff
@@ -0,0 +1,4 @@
+from os import path
```

Comment:
this file gets a veto fro me, neither of these should be constant, what if the user wants to enter a different output folder? This makes some very dangerous assumptions about the file structure that will cause us misery down the line

Reply:
these files aren't mine, they are from develop (had these open already before i switched to the new matrix branch and pulling from dev-4.0).